app/views/schedules/show.xml.haml

```diff
@@ -16,7 +17,7 @@
           %room{ name: room.name }
             - events_in_rooms[room].each do |event|
               %event{ guid: event.guid, id: event.id }
-                %date= event.time.iso8601
+                %date= event.time.iso8601.tr('Z', '')
```

Comment:
@differentreality like this https://github.com/openSUSE/osem/issues/1188#issuecomment-287754919 ?